src/Combobox/index.stories.js

```diff
@@ -150,6 +150,82 @@ export const InADialog = () => {
   );
 };
 
+export const WithCategories = Template.bind({});
+WithCategories.args = {
+  id: "withCategories",
+  label: "Transfer to",
+  children: [
+    <Combobox.Category label="Checking">
+      <Combobox.Item searchValue="Business Checking" value="checking1">
+        Business Checking - 11234
+      </Combobox.Item>
+      <Combobox.Item searchValue="Main Checking" value="checking2">
+        Main Checking - 13989
+      </Combobox.Item>
+      <Combobox.Item searchValue="Joint Checking" value="checking3">
+        Joint Checking - 14857
+      </Combobox.Item>
+    </Combobox.Category>,
+    <Combobox.Category label="Savings">
+      <Combobox.Item searchValue="Business Checking" value="savings1">
```

Comment:
Minor, but think there's some copy/pasta here. Think you meant "Business Savings"